packages/components/src/components/dropdown/dropdown.js

```diff
@@ -39,6 +40,11 @@ class Dropdown extends mixin(
    */
   constructor(element, options) {
     super(element, options);
+    this.triggerNode = this.element.querySelector(this.options.selectorTrigger);
```

Comment:
Our codebase doesn't keep node refs as instance properties. Good to keep doing the same unless we make a decision to make the change for the entire codebase.

Reply:
@asudoh Sorry I'm not sure what you are asking here. Are you saying i should query for the elements in each function that I use them?

Reply:
You are right @elizabethsjudd!